doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -504,6 +504,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-9-0 v2-8-0 v2-7-0
+
+KRKNWK-19915: Total operational hours are saved to NVM with interval defined by :kconfig:option:`CONFIG_CHIP_OPERATIONAL_TIME_SAVE_INTERVAL` Kconfig option.
```

Comment:
```suggestion
KRKNWK-19915: Total operational hours are saved to NVM with interval defined by the :kconfig:option:`CONFIG_CHIP_OPERATIONAL_TIME_SAVE_INTERVAL` Kconfig option
```
The title line always without a full stop. And I forgot "the" on the first round.

Reply:
thankd, I've applied change locally and will push it with update of manifest file